### PR TITLE
Style new nav

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Style new nav
Update homepage copy

## QA

- Run ./run then go to http://0.0.0.0:8015
- Check the nav in l/m and s it should be full width in small
- Check the homepage copy is the same as the doc 
- UX has seen the demo and approved

## Details

Doc: https://docs.google.com/document/d/1bvQDL9UlLMx8DFNQTCELPB4I3qBNwSV-OAig7z5-Jts/edit

### CODE REVIEW

**Comment by grahambancroft** [comment]
[Style new nav](https://trello.com/c/j5rad2Or/3238-style-new-nav)

File: templates/pages/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ insights_feed:
 tutorial_feed: true
 ----
 
-# Millions of developers use Ubuntu to write, build and deploy their code
+# Use Ubuntu to write, build and deploy your code
```
Comment (anthonydillon) [inline templates/pages/index.md:10]: There is a comment in the copy doc asking for this title to be removed.

**Review by anthonydillon — APPROVED** [review]
Code and QA is good. With a question on the title.

File: templates/pages/index.md (hunk 1 above)
Comment (grahambancroft) [reply templates/pages/index.md:10]: It says "please revise heading”, not remove

**Pull request merged**